src/Microsoft.OpenApi/Any/OpenApiByte.cs

```diff
@@ -6,12 +6,20 @@ namespace Microsoft.OpenApi.Any
     /// <summary>
     /// Open API Byte
     /// </summary>
-    public class OpenApiByte : OpenApiPrimitive<byte>
+    public class OpenApiByte : OpenApiPrimitive<byte[]>
     {
         /// <summary>
         /// Initializes the <see cref="OpenApiByte"/> class.
         /// </summary>
         public OpenApiByte(byte value)
+            : this(new byte[] { value })
+        {
+        }
+
```

Comment:
Remove this

Reply:
Remove this is a breaking change. We should avoid breaking change.

Reply:
Yes, but this is technically a bug fix since our old model is incorrect

---
In reply to: [265768029](https://github.com/Microsoft/OpenAPI.NET/pull/391#discussion_r265768029) [](ancestors = 265768029)

Reply:
`public OpenApiByte(byte value)` is not a technically bug. Keeping it also provides a method for customer to use.

Reply:
I guess that's fine. 

---
In reply to: [265771924](https://github.com/Microsoft/OpenAPI.NET/pull/391#discussion_r265771924) [](ancestors = 265771924)